src/typings/targetDescriptor.types.ts

```diff
@@ -171,6 +171,16 @@ export interface ContainingScopeModifier {
   scopeType: ScopeType;
 }
 
+export interface PreviousContainingScopeModifier {
+  type: "previousContainingScope";
+  scopeType: ScopeType;
+}
+
+export interface NextContainingScopeModifier {
+  type: "nextContainingScope";
+  scopeType: ScopeType;
+}
```

Comment:
Is having these as separate interfaces useful? Maybe we can simplify this to be a single interface. Something like this

```ts
export interface SiblingContainingScopeModifier {
  type: "previousContainingScope" | "nextContainingScope";
  scopeType: ScopeType;
}
```